frontend/src/ProposalMetadata.elm

```diff
@@ -89,18 +89,53 @@ bodyDecoder =
 
 
 {-| JSON decoder for author witness.
+Supports three formats:
+
+1.  Name-only: { "name": "..." } - for preconfigured authors
+2.  Nested witness: { "name": "...", "witness": { "witnessAlgorithm": "...", "publicKey": "...", "signature": "..." } }
+3.  Flat witness: { "name": "...", "witnessAlgorithm": "...", "publicKey": "...", "signature": "..." }
+
 -}
 authorWitnessDecoder : JD.Decoder AuthorWitness
 authorWitnessDecoder =
```

Comment:
Let’s not do 3 different formats here, we should only support one format which has the shape of the one in the CIP: 
```json
  "authors": [
    {
      "name": "Ryan Williams",
      "witness": {
        "witnessAlgorithm": "CIP-0008",
        "publicKey": "7ea09a34...37480a",
        "signature": "84582a...71409"
      }
    }
```
We can require that the environment variable to provide pre-defined authors just provide the name anyway, without witness, because the full witness will be generated by the signature creation.